codespell_lib/data/dictionary.txt

```diff
@@ -28898,6 +28908,7 @@ implemetning->implementing
 implemetns->implements
 implemets->implements
 implemnt->implement
+implemnt->implement
```

Comment:
Isn't the above a duplicate? No worries, that's the perfect oocasion to add all verbal forms:
```suggestion
implemntation->implementation
implemnted->implemented
implemnting->implementing
implemnts->implements
```